os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_central/ble_tizenrt_client.c

```diff
@@ -118,7 +291,7 @@ void scan_stop_cb(void *arg)
 	} while(new_state.gap_scan_state != GAP_SCAN_STATE_IDLE);
 }
 
-trble_result_e rtw_ble_client_start_scan_with_filter(trble_scan_filter* scan_parm)
+trble_result_e rtw_ble_client_start_scan_with_filter(trble_scan_filter* scan_parm, bool whitelist_enable)
```

Comment:
I have a query.
If I enable 'whitelist_enable' without adding any addresses, what will be happened?
All results come up? or nothing comes up?

Reply:
When whitelist filter is enable without adding any addresses is added in white list, nothing will come out.